### ISSUE TITLE
Overflow: hidden on Webview's container causes Android 9 to crash

### ISSUE THREAD

**Issue by AdamZaczek** [react-native-community/react-native-webview#448]
When webview is being loaded inside an item from FlatList there is an issue where it overflows from its container effectively covering the whole screen.
There seems to be a popular fix for this - using overflow: hidden on webview's container.
Sadly, this solution causes the app to crash on a physical device: Samsung Galaxy S9+ with Android 9.
It works perfectly well on other devices and simulators created in android studio.

I stumbled upon another possible solution: to route between navigation stacks after the screen with webview is completely loaded.
But it's far from ideal as it's prone to bugging out - the issues reappears sometimes and is extremely messy to implement.

**Comment by Titozzz** [comment]
duplicate of #429.

TLDR: https://bugs.chromium.org/p/chromium/issues/detail?id=944020#c17 will fix it

**Comment by malonguwa** [comment]
@AdamZaczek hey just wondering if your Samsung Galaxy S9+ with Android 9 crash issue got fixed in Android Chrome latest release?

need to upgrade the android chrome to the latest 73.03683.90 version. (the buggy version is 73.0.3683.75).

Have you tried this walk around?

<View style={{..., overflow:'hidden'}}>
<WebView
    .......,
    style={{flex:1, opacity:0.99}}
/>
</View>

This is working in my S8 and Pixel real device, but I do not have S9 to test it.

**Comment by AdamZaczek** [comment]
New chrome version works perfectly on my device @malonguwa .
